### PR TITLE
FPL-217: Add new users

### PR DESCRIPTION
**Pull request by kurt-uk**
### Change description ###
added new users and updated the tests to reflect the log in of the new user, issues with ccd and solicitors mean that they dont have an event summary page so methods that checked these fields were removed

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: ccd-definition/FixedLists.json [hunk 1]
```diff
@@ -32,8 +32,14 @@
   {
     "LiveFrom": "01/01/2017",
     "ID": "AuthorityFixedList",
-    "ListElementCode": "example",
-    "ListElement": "Example Local Authority"
+    "ListElementCode": "swansea",
```
Comment (swalker125) [inline ccd-definition/FixedLists.json]: ListElementCode needs to follow tech plan codes https://tools.hmcts.net/confluence/display/RPL/Notify+on+Submission

Swansea is SA, Hillingdon is HN

File: docker-compose.yml [hunk 2]
```diff
@@ -15,7 +15,7 @@ services:
       - CORE_CASE_DATA_API_URL=http://ccd-data-store-api:4452
       - IDAM_S2S_AUTH_URL=http://service-auth-provider-api:8080
       - IDAM_S2S_AUTH_TOTP_SECRET=AABBCCDDEEFFGGHH
-      - FPL_LOCAL_AUTHORITY_NAME_MAPPING=example.com=>example
+      - FPL_LOCAL_AUTHORITY_NAME_MAPPING=swansea.gov.uk=>swansea;hillingdon.gov.uk=>hillingdon
```
Comment (swalker125) [inline docker-compose.yml]: Needs to follow the tech plan codes

File: kubernetes/deployment.template.yaml [hunk 3]
```diff
@@ -38,7 +38,7 @@ data:
   IDAM_S2S_AUTH_URL: "http://service-auth-provider-api"
   IDAM_S2S_AUTH_TOTP_SECRET: "AABBCCDDEEFFGGHH"
   IDAM_API_URL: "http://idam-api"
-  FPL_LOCAL_AUTHORITY_NAME_MAPPING: "example.com=>example"
+  FPL_LOCAL_AUTHORITY_NAME_MAPPING: "swansea.gov.uk=>swansea;hillingdon.gov.uk=>hillingdon"
```
Comment (swalker125) [inline kubernetes/deployment.template.yaml]: needs to be changed to follow tech plan

File: docker-compose.yml (hunk 2 above)
Comment (kurt-uk) [reply docker-compose.yml]: -_- asked you about that....

**Comment by SamanthaSJ** [comment]
Roles have been verified in this PR. Happy for this to be merged to master.

**Pull request merged**